fedimint-server/src/net/api.rs

```diff
@@ -575,14 +574,6 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConsensusApi>> {
                 Ok(fedimint.get_guardian_config_backup(password).await?)
             }
         },
-        api_endpoint! {
-            VERIFY_CONFIG_HASH_ENDPOINT,
-            ApiVersion::new(0, 0),
-            async |fedimint: &ConsensusApi, context, _v: ()| -> BTreeMap<PeerId, sha256::Hash> {
-                check_auth(context)?;
-                Ok(get_verification_hashes(&fedimint.cfg))
-            }
-        },
```

Comment:
@okjodom @Kodylow Why did we add that endpoint to the running server API originally? Was it to prevent timing issues when some guardians started consensus before others? I can't see the point because I assume this API is only used by the guardian running this instance, so isn't needed after consensus is running, but want to be sure.

Reply:
The endpoint requires auth, hence can only be requested by the corresponding guardian via CLI or UI; however this guardian already explicitly started the consensus so why would he check the verification hashes now?

This is in contrast to the SERVER_CONFIG_CONSENSUS_ENDPOINT which also returns a config hash when running, however this is supposed to be requested by the other guardians to check against their own.